File: Python/uri_1072.py
n = int(input())
v = []
IN = 0
OUT = 0
for i in range(0, n):
    v.append(int(input()))
    if 10 <= v[i] <= 20:
        IN += 1
    else:
        OUT += 1
print('{} in'.format(IN))
print('{} out'.format(OUT))

File: Go/uri_1072.go
package main

import "fmt"

func main() {
	var n, in, out = 0, 0, 0
	fmt.Scanln(&n)
	v := make([]int, n)
	for _, i := range v {
		fmt.Scanln(&v[i])
		if v[i] >= 10 && v[i] <= 20 {
			in++
		} else {
			out++
		}
	}
	fmt.Printf("%d in\n%d out\n", in, out)
}
